lib/pdf-editor-view.js

```diff
@@ -251,6 +250,16 @@ export default class PdfEditorView extends ScrollView {
     this.container.find("canvas").remove();
     this.canvases = [];
 
+    let reverseSyncClicktype = null
+    switch(atom.config.get('pdf-view.reverseSyncBehaviour')) {
```

Comment:
What happens if I change the config value after the PDF was loaded? For example, by default, the value is "Click", so that's what's used when I load a PDF and `reverseSyncClicktype` is set to `click`. After the PDF loads, I go to the Settings View and select "Double Click" instead. In that case, I'd expect that only a double click triggers the reverse sync behavior. Instead, a single click still triggers the behavior. That's because we're not listening on the change of this config setting and `reverseSyncClicktype` is still `click`. I think we should be listening for changes of that config setting and updating the canvas click and double-click handlers accordingly. 

Let me know if you disagree or if I'm missing something, @nsaje